### PR TITLE
Doc tests

### PR DESCRIPTION
**Pull request by AmitShaul**
## Motivation
a more understandable code

### CODE REVIEW

**Comment by AmitShaul** [comment]
bors try

File: tests/ed25519/eddsa.py [hunk 1]
```diff
@@ -1,7 +1,14 @@
 from pure25519.eddsa import H, Hint
-from pure25519.basic import (bytes_to_clamped_scalar, bytes_to_element, bytes_to_scalar, scalar_to_bytes, Base, L)
+from pure25519.basic import bytes_to_clamped_scalar, bytes_to_element, bytes_to_scalar, scalar_to_bytes, Base, L
 from pure25519.eddsa import create_signing_key, create_verifying_key
 
+"""
+This file, using pure25519 package, provides high security,
+generating key pers and signatures for the miners.
```
Comment (noamnelke) [inline tests/ed25519/eddsa.py]:
```suggestion
generating key pairs and signatures for the miners.
```

File: tests/queries.py [hunk 2]
```diff
@@ -168,6 +168,22 @@ def poll_query_message(indx, namespace, client_po_name, fields, findFails=False,
 
 def query_message(indx, namespace, client_po_name, fields, find_fails=False, start_time=None, queries=None,
                   is_print=True):
+    """
+    Send a query and return its' hits, this function is used by most of the query functions.
```
Comment (noamnelke) [inline tests/queries.py]:
```suggestion
    Send a query and return its hits, this function is used by most of the query functions.
```

File: tests/tx_generator/models/wallet_api.py [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
 class WalletAPI:
     """
-    WalletAPI communicates with a random miner pod for
+    WalletAPI communicates with a [random] miner pod for
```
Comment (noamnelke) [inline tests/tx_generator/models/wallet_api.py:11]: what does the `[...]` mean?

**Review by noamnelke — APPROVED** [review]

File: tests/tx_generator/models/wallet_api.py (hunk 3 above)
Comment (AmitShaul) [reply tests/tx_generator/models/wallet_api.py:11]: optional

File: tests/ed25519/eddsa.py (hunk 1 above)
Comment (AmitShaul) [reply tests/ed25519/eddsa.py]: fixed in 88127f439bd8e20b30ac26cb0f739a8059935186

File: tests/queries.py (hunk 2 above)
Comment (AmitShaul) [reply tests/queries.py]: fixed in 88127f439bd8e20b30ac26cb0f739a8059935186

**Comment by noamnelke** [comment]
Merged as part of **Consolidated linting (#1958)**

**Pull request closed without merging**